sync-core/src/test/java/com/cloudant/sync/replication/BasicPullStrategyMockTest.java

```diff
@@ -159,4 +226,22 @@ public void error(ReplicationStrategyErrored re) {
         }
     }
 
+    private List<DocumentRevs> getMockedRevs(String fixturePath) throws Exception{
```

Comment:
Not a very descriptive name, suggest calling it `loadOpenRevsResponseFromFixture`.